### PR TITLE
Add application flags and ID to the ready event.

### PR DESCRIPTION
**Pull request by FasterSpeeding**
### Summary
Add application flags and ID to the ready event.

### Checklist

- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

File: hikari/applications.py [hunk 1]
```diff
@@ -58,6 +59,43 @@
...
+    """Denotes that a verified application can use the GUILD_MEMBERS intent."""
+
+    GUILD_MEMBERS_INTENT = 1 << 15
+    """Denotes that the application has the GUILD_MEMBERS intent enabled in it's dashboard."""
+
+    VERIFICATION_PENDING_GUILD_LIMIT = 1 << 16
+    """Denotes that the application's verification is pending."""
+
+    EMBEDDED = 1 << 17
+    # TODO: what does this mean
```
Comment (davfsa) [inline hikari/applications.py:96]: Before merging this I would like to find out what this actually means

Comment (FasterSpeeding) [reply hikari/applications.py:96]: The feature hasn't been released yet so there's no reference to document it against

Comment (davfsa) [reply hikari/applications.py:96]: Then I think its best to wait and see what gets documented

Comment (FasterSpeeding) [reply hikari/applications.py:96]: Since they don't merge most prs anymore that could mean waiting for years

Comment (davfsa) [reply hikari/applications.py:96]: Could you add this?

```suggestion
    """Denotes that the application is embedded."""
```

or maybe


```suggestion
    """Denotes that the application is embedded into Discord."""
```
as this is probably what it means

**Review by davfsa — APPROVED** [review]

**Pull request merged**